### PR TITLE
LPD-24671 Prevent re-authentication

### PR DESCRIPTION
**Pull request by istvansajtos**
[LPD-24671](https://liferay.atlassian.net/browse/LPD-24671)
Hi,

Please review my changes and let me know if you have any concern.

Thank you!
István

cc @stian-sigvartsen @zsigmondrab

### CODE REVIEW

**Comment by istvansajtos** [comment]
ci:help

**Comment by istvansajtos** [comment]
ci:test:relevant

**Comment by istvansajtos** [comment]
ci:test:oidc

File: modules/apps/login/login-authentication-openid-connect-web/src/main/java/com/liferay/login/authentication/openid/connect/web/internal/portlet/action/OpenIdConnectLoginRequestMVCActionCommand.java [hunk 1]
```diff
@@ -106,6 +108,20 @@ public void doProcessAction(
 		throws Exception {
 
 		try {
+			ThemeDisplay themeDisplay =
+				(ThemeDisplay)actionRequest.getAttribute(WebKeys.THEME_DISPLAY);
+
+			if (themeDisplay.isSignedIn()) {
+				String redirect = ParamUtil.getString(
+					actionRequest, "redirect");
```
Comment (stian-sigvartsen) [inline modules/apps/login/login-authentication-openid-connect-web/src/main/java/com/liferay/login/authentication/openid/connect/web/internal/portlet/action/OpenIdConnectLoginRequestMVCActionCommand.java:116]: @istvansajtos we need to escape the redirect value before using it. Can you add a commit?

**Comment by istvansajtos** [comment]
ci:test:sf

**Comment by istvansajtos** [comment]
ci:test:relevant

**Comment by istvansajtos** [comment]
ci:test:oidc

**Comment by istvansajtos** [comment]
@stian-sigvartsen , I've added the requested change. Thank you!

**Comment by stian-sigvartsen** [comment]
Looks good @istvansajtos !

**Comment by stian-sigvartsen** [comment]
ci:forward

**Pull request closed without merging**